apps/core/priv/repo/migrations/20170909040532_create_widgets.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Core.Repo.Migrations.CreateWidgets do
+  use Ecto.Migration
+
+  def up do
+    create table(:widgets, primary_key: false) do
+      add :id, :uuid, primary_key: true
+      add :name, :string
+      add :slug, :string
```

Comment:
Not sure we'll need slug. May've mislead you from the UI JSON.. I can envisage a Dashboard needing custom URLs to share but not Widgets?